src/react-apps/applications/ux-editor/src/containers/FormFiller.tsx

```diff
@@ -18,16 +18,34 @@ export interface IFormFillerProps {
   designMode: boolean;
   formDataCount: number;
   language: any;
+  workflowStep: WorkflowSteps;
 }
 
-export interface IFormFillerState { }
+export interface IFormFillerState {
+  workflowStep: WorkflowSteps;
+}
 
 /**
  * Component responsible for rendering the layout around the
  * form itself.
  */
 export class FormFillerComponent extends React.Component<IFormFillerProps, IFormFillerState> {
 
+  constructor(props: IFormFillerProps, state: IFormFillerState) {
+    super(props, state);
+    this.state = {
+      workflowStep: props.workflowStep,
+    };
+  }
+
+  public componentWillReceiveProps(nextProps: IFormFillerProps) {
```

Comment:
`componentWillReceiveProps` should be switched out here as well

Reply:
RIght. Replaced with getDerivedStateFromProps.